plugin.program.newgrounds/changelog.txt

```diff
@@ -0,0 +1,11 @@
+1.0.5 Changed from cURL to JSON API
```

Comment:
Changelog is no longer provided on the interface. Please use the `<news>` tag in addon.xml